packages/rpc-core/src/index.ts

```diff
@@ -282,98 +281,114 @@ export default class Rpc implements RpcInterface {
     );
   }
 
-  private formatOutput (method: RpcMethod, params: Codec[], result?: any): Codec | (Codec | null | undefined)[] {
-    const base = createTypeUnsafe(method.type, [result]);
+  private treatAsHex (key: StorageKey): boolean {
+    // :code is problematic - it does not have the length attached, which is
+    // unlike all other storage entries where it is indeed properly encoded
+    return ['0x3a636f6465'].includes(key.toHex());
+  }
```

Comment:
We have basically moved the check for . this from `Bytes` (where we try an determine what we are working with), to here where we are being explicit in what the "problematic" keys are